.github/workflows/build-and-deploy.yml

```
@@ -0,0 +1,47 @@
+name: Build and deploy
+on: 
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Cache the dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
+
+      - name: Install Dependencies
+        run: npm install
+
+      - name: Lint
+        run: npm run lint
+
+      - name: Rename config files
+        run: | 
+          cp config/env/_common.js config/env/common.js |
+          cp config/env/_development.js config/env/development.js |
+          cp config/env/_production.js config/env/production.js
+
+      - name: Build
+        run: npm run build
+
+      - name: Move dist to AWS
+        run: |
+          aws s3 sync ./dist s3://test.theaterhub.ro --acl public
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_DEFAULT_REGION: ${{ secrets.AWS_REGION }}
```

Newline at end of file
```suggestion
          AWS_DEFAULT_REGION: ${{ secrets.AWS_REGION }}

```